__tests__/lib/commands.test.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable jest/no-conditional-expect */
```

Comment:
Could we disable it for each line or is it affecting the whole file?

Reply:
Every test in this one has a conditional expect unfortunately.